build.gradle

```
@@ -94,7 +94,7 @@ ext {
     libaddressinputVersion = '0.0.2'
     eventBusVersion = '3.2.0'
     googlePlayCoreVersion = '1.10.0'
-    coroutinesVersion = '1.3.6'
+    coroutinesVersion = '1.4.3-native-mt'
```

I think we should eventually change it to not `native-mt` version as `native-mt` is about Kotlin Multiplatform threading on Native target.

```suggestion
    coroutinesVersion = '1.4.3'
```